docs/custom_text_ops.md

```diff
@@ -11,11 +11,11 @@
 |StringRegexReplace| Supported  |
 |StringSplit | Supported       |
 |StringUpper  | Supported     |
-|StringSlice | Under development|
-|StringLength | Under development |
+|StringLength | Supported |
+|StringConcat | Supported |
```

Comment:
And there is Concat for String Tensor in ONNX spec.

https://github.com/onnx/onnx/blob/master/docs/Operators.md#concat